docarray/array/storage/weaviate/seqlike.py

```diff
@@ -54,9 +54,9 @@ def __contains__(self, x: Union[str, 'Document']):
         else:
             return False
 
-    def __del__(self):
+    def sync(self):
         """Delete this :class:`DocumentArrayWeaviate` object"""
-        super().__del__()
+        super().sync()
```

Comment:
Why does it not think these are covered by tests? Isn't tests/unit/array/test_advance_indexing.py::test_offset2ids_persistence explicitly calling sync on this DocumentStore?

Reply:
Oh its not being overriden... thanks codecov <3